<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-K1C0HJ9" ignored="0" name="assertion test" tests="1" failures="0" timestamp="2020-10-01T23:26:00 IST" time="0.176" errors="0">
  <testcase name="@BeforeTest init" time="0.025" classname="academy.AssertionExample">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at resources.DriverClass.initialiseDriver(DriverClass.java:68)
at academy.AssertionExample.init(AssertionExample.java:26)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@AfterTest destroy" time="-1.601574960877E9" classname="academy.AssertionExample">
    <skipped/>
  </testcase> <!-- @AfterTest destroy -->
  <testcase name="checkText" time="0.0" classname="academy.AssertionExample">
    <skipped/>
  </testcase> <!-- checkText -->
</testsuite> <!-- assertion test -->
